data/json/monsters/insect_spider.json

```
@@ -3050,7 +3050,7 @@
     "vision_night": 5,
     "harvest": "arachnid",
     "dissect": "dissect_troglobite_sample_single",
-    "upgrades": { "age_grow": 12, "into": "mon_antlion_giant" },
+    "upgrades": { "half_life": 42, "into": "mon_antlion_giant" },
```

```suggestion
    "upgrades": { "age_grow": 42, "into": "mon_antlion_giant" },
```
Turning this into an evo means instead of nonthreatening adults littering the landscape with annoying/dangerous grubs the grubs go extinct and the adults proliferate.